docs/konnectors.md

```diff
@@ -320,18 +320,40 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Konnector install and run details
+
+### Install
+
+The konnectors will be installed in the .cozy_konnectors directory (not in VFS ?) using npm install
```

Comment:
Do we use `yarn` or `npm` to install it? If `npm`, is it installed with the `-g` flag, or do we need to create a stub `package.json`?